SU2_CFD/include/CMarkerProfileReaderFVM.hpp

```diff
@@ -155,6 +156,17 @@ class CMarkerProfileReaderFVM {
     return numberOfColumnsInProfile[val_iProfile];
   }
   
+    /*!
+   * \brief Get the number of columns of data in a profile if an interpolation method was used (multigrid).
+   * \param[in] val_iProfile - current profile index.
+   * \param[in] Interpolate - bool value for Interpolation
+   * \returns Number of columns of data in the (if) interpolated profile.
+   */
+  inline unsigned long GetNumberOfColumnsInProfile(int val_iProfile, bool Interpolate) {
+    if (Interpolate == true) {return numberOfColumnsInProfile[val_iProfile]+dimension;}
+    else {return numberOfColumnsInProfile[val_iProfile];}
+  }
```

Comment:
This class should have no need to know about interpolation aspects, we want everything to be as orthogonal as possible, if you need to correct the number of columns do it on the call site.

Reply:
done